applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -137,16 +139,24 @@ def _ComputeLiftFromJumpCondition(self):
     def _ComputeLiftFromJumpCondition3D(self):
 
         potential_integral = 0.0
+        drag_integral = 0.0
         for cond in self.trailing_edge_model_part.Conditions:
             length = cond.GetGeometry().Area()
             for node in cond.GetNodes():
                 potential = node.GetSolutionStepValue(CPFApp.VELOCITY_POTENTIAL)
                 auxiliary_potential = node.GetSolutionStepValue(CPFApp.AUXILIARY_VELOCITY_POTENTIAL)
+                velocity = node.GetValue(KratosMultiphysics.VELOCITY)
```

Comment:
In order to retrieve the nodal velocity one has to use the `compute_nodal_value_process` first right?

I would either add a check and throw a warning if it hasn't been run beforehand or directly call it automatically inside the `_ComputeLiftFromJumpCondition3D()` (I prefer the second option)

Reply:
You are right, Ill go for the second option then :+1: